src/test/java/org/folio/service/finance/FiscalYearServiceTest.java

```diff
@@ -10,34 +10,54 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import java.lang.reflect.Field;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.UUID;
 import java.util.concurrent.CompletionException;
+import java.util.concurrent.ExecutionException;
 
+import org.folio.rest.core.models.RequestEntry;
+import org.folio.rest.core.RestClient;
+import io.vertx.junit5.VertxExtension;
+import org.folio.service.finance.FundService;
 import org.folio.rest.acq.model.finance.FiscalYear;
+import org.folio.service.finance.FiscalYearService;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
```

Comment:
I think there is import style issue because static part should be in the beginning, please reformat again